quickstarts/08-data-agent-mcp-chainlit/requirements.txt

```diff
@@ -0,0 +1,4 @@
+dapr-agents>=0.5.1
+python-dotenv
+chainlit
+psycopg
```

Comment:
Even after `pip install -r requirements.txt`, I had to install `psycopg[binary]` ;) . Can we add that to the requirements.txt please?

Reply:
Added